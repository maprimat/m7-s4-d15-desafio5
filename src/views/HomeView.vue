<template>
    <div class="home">
        <img alt="Vue logo" src="../assets/logo.png">
        <PostsView />
    </div>
</template>

<script>
import PostsView from './PostsView.vue';

export default {
    name: 'HomeView',
    components: {
        PostsView
    }
}
</script>
