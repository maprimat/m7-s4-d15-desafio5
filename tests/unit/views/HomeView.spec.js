import HomeView from "@/views/HomeView.vue";
import { shallowMount } from "@vue/test-utils";

describe("HomeView", () => {
    test("Se valida snapshot de estructura HTML: HomeView", () => {
        
        //Selección el Sujeto de pruebas
        const wrapper = shallowMount(HomeView);

        //Aserción
        expect(wrapper.html()).toMatchSnapshot(`
        <div class="home">
            <img alt="Vue logo" src="../assets/logo.png">
            <PostsView />
        </div>
    `);
    });
});