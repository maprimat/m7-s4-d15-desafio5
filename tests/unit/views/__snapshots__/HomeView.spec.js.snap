// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeView Se valida snapshot de estructura HTML: HomeView: 
        <div class="home">
            <img alt="Vue logo" src="../assets/logo.png">
            <PostsView />
        </div>
     1`] = `
<div class="home"><img alt="Vue logo" src="">
  <posts-view-stub></posts-view-stub>
</div>
`;
